data/json/items/software.json

```
@@ -80,5 +80,50 @@
     "name": { "str": "integrated circuit datasheet archives", "str_pl": "misc software" },
     "description": "Huge archives of numerous IC circuit datasheets from several major manufacturers.  Probably valuable to the right person, as it would be hard to salvage and reuse these components without them.",
     "price": 40000
+  },
+  {
+    "abstract": "software_tool",
+    "type": "TOOLMOD",
+    "flags": [ "SOFTWARE" ],
+    "container": "usb_drive",
+    "//": "note: str_pl = str because otherwise you get 'USB drives with productivity softwares'",
+    "//": "- account for this on all software items because that's how they'll be found in the wild",
+    "name": { "str": "usable software", "str_pl": "usable software" },
```

```suggestion
    "name": { "str_sp": "usable software" },
```
where `sp` is singular/plural. Used when plural = singular.